misc/CFGs/CFG.txt

```diff
@@ -0,0 +1,87 @@
+**************************************************************
+CFG
+**************************************************************
+
+Program --> Commands $
+
+Commands --> FunctionDcl Commands
+         | VarDcl Commands
+         | VarAssign Commands
+         | Expression Commands
+         | EPSILON
+
+Operator --> bracketL
+           | bracketR
+           | parenL
+           | parenR
+           | curlyL
+           | curlyR
+           | assign
+           | functionPassOp
+           | arrow
+
+
+Type --> number
+       | text
+       | tuple
+       | list
+       | dict
+       | none
+       | func
+       | var
+
+Expression --> FunctionCall
+             | Value
+
+Value --> Literal
+        | identifier
+
+Literal --> literalNumber
+          | literalText
+          | literalTuple
+          | literalList
+          | literalDict
+          | LambdaExpression
+
+Statements --> Statement linebreak Statements
+             | EPSILON
+
+Statement --> VarDcl
+            | VarAssign
+            | Expression
+            | ReturnStatement
+
+VarDcl --> Type identifier
+         | Type VarAssign
+
+VarAssign --> identifier assign Expression
+
+FunctionDcl --> FunDclStart identifier assign LambdaParams curlyL Statements curlyR
+
+FunDclStart --> func
+              | var
+
+FunDclParamList --> FunDclParameter FunDclSecondaryParameter
+                  | EPSILON
+
+FunDclSecondaryParameter --> comma FunDclParameter FunDclSecondaryParameter
+                           | EPSILON
+
+FunDclParameter --> Type identifier
+
+FunctionCall --> identifier
+               | FunCallParameter identifier FunCallParamList
+
+FunCallParamList --> FunCallParameter FunCallParameterList
+                   | parenL FunCallParameter FunCallParameterList parenR
+                   | EPSILON
+
+FunCallParameter --> Value
+                   | functionPassOp identifier
```

Comment:
A function can be parsed by just using an identifer - functionPassOp is not needed.